### PR TITLE
Added post-release Helm test

### PR DESCRIPTION
**Pull request by solonish**
Signed-off-by: Nishanth Solomon <solonish@amazon.com>

### CODE REVIEW

File: e2e/helm_test.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+  POD_NAME=$(kubectl get pods -n $K8S_NAMESPACE -ojson | jq -r ".items[0].metadata.name")
+
+  if [ -z "$POD_NAME" ]; then
+    echo "[ERROR] Found empty ACK controller pod name. Exiting ..."
+    exit 1
+  fi
+
+  POD_STATUS=$(kubectl get pod/"$POD_NAME" -n $K8S_NAMESPACE -ojson | jq -r ".status.phase")
+  [[ $POD_STATUS == Pending ]] && echo "pod status is $POD_STATUS . Waiting ... " && sleep 10
```
Comment (anbaig) [inline e2e/helm_test.sh:85]: You could probably use `kubectl wait` here and wait for the pod to reach a ready status

File: e2e/helm_test.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+  if echo "$LOGS" | grep -q "ERROR"; then
+    echo "[ERROR] Found following ERROR statements in controller logs."
+    print_line_separation
+    echo "$LOGS" | grep "ERROR"
+    print_line_separation
+    echo "[ERROR] Exiting ..."
+    exit 1
+  fi
+  echo "ok."
```
Comment (anbaig) [inline e2e/helm_test.sh:104]: remove

File: e2e/helm_test.sh [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+  echo -n "uninstalling the Helm Chart $HELM_CHART_NAME in namespace $K8S_NAMESPACE ... "
+  helm uninstall --namespace "$K8S_NAMESPACE" "$DEPLOYMENT_NAME" 1>/dev/null || exit 1
+  echo "ok."
```
Comment (anbaig) [inline e2e/helm_test.sh:108]: remove

Comment (solonish) [reply e2e/helm_test.sh]: Removed echo "ok."

File: e2e/helm_test.sh (hunk 2 above)
Comment (solonish) [reply e2e/helm_test.sh]: Removed echo "ok."

File: e2e/helm_test.sh (hunk 1 above)
Comment (solonish) [reply e2e/helm_test.sh]: Changed to kubectl wait

**Review by anbaig — APPROVED** [review]

**Pull request merged**